### ISSUE TITLE
Contact Book (Feature Request)

### ISSUE THREAD

**Issue by sdmeier** [ACCORD5/TrellisDesk#20]
This is a feature request.  
I would love to see a Contact Book to hold information such as Company Names,  Phone Numbers, Email addresses, Postal Address...

Thanks for your consideration.

**Comment by DanielRuf** [comment]
I would also like to see something like this as contact book: http://datatables.net/  could be useful. 

We should collect more suggestions so we can develop it faster.

`td_contacts` would be the right db tabl i think

so which db table fields do we need?

**Comment by djtarazona** [comment]
Good idea, however I feel this might better serve as an addon.  We don't have any sort of addon framework for Trellis Desk right now so I'm not sure where this will go, but we'll get to it eventually!
